assignments/hackyourtemperature/server.js

```diff
@@ -0,0 +1,29 @@
+import express from 'express';
+
+const app = express(); 
+
+
+app.use(express.json()); 
+
+
+app.get('/', (req, res) => {
+  res.send('hello from backend to frontend!');
+});
+
+
+app.post('/weather', (req, res) => {
+
+  const { cityName} = req.body;

```

Comment:
Small suggestion to add a space so your code is easier to read.

```suggestion
  const { cityName } = req.body;
```

Usually on bigger projects, you have tools to format automatically your code, if you are curious you can lookup what a "linter" is and more specifically search for a linter for javascript (like prettier)